### PR TITLE
Mark cache variables as advanced

### PR DESCRIPTION
**Pull request by ffoulkes**
- Set the ADVANCED attribute on cache variables that should not normally be displayed in cmake GUIs.

### CODE REVIEW

File: cmake/FindOVS.cmake [hunk 1]
```diff
@@ -66,6 +75,7 @@ elseif(EXISTS "${OVS_INCLUDE_DIR}/openvswitch/version.h")
     string(REGEX REPLACE
         "[^0-9.]+([0-9.]+).*" "\\1" OVS_VERSION ${_version_string})
     set(OVS_VERSION ${PC_OVS_VERSION} CACHE STRING "OVS version")
+    mark_as_advanced(OVS_VERSION)
```
Comment (nupuruttarwar) [inline cmake/FindOVS.cmake:78]: can OVS_VERSION be useful information to be displayed?

**Review by nupuruttarwar — APPROVED** [review]
LGTM

File: cmake/FindOVS.cmake (hunk 1 above)
Comment (ffoulkes) [reply cmake/FindOVS.cmake:78]: The OVS version is already being displayed, as are the versions of the other "found" packages:

```bash
dfoster@dgfoster-mobl1:~/latest$ cmake -B build -DOVS_INSTALL_DIR=ovs/install
-- The C compiler identification is GNU 9.4.0
-- The CXX compiler identification is GNU 9.4.0
      .
      .
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.1")
      .
      .
-- Checking for module 'libdpdk'
--   Found libdpdk, version 22.11.0-rc0
-- Checking for module 'libopenvswitch'
--   Found libopenvswitch, version 3.0.90
-- Found OVS: /home/dfoster/latest/ovs/install/lib/libopenvswitch.a (found version "3.0.90")
-- Checking for module 'libnl-3.0'
--   Found libnl-3.0, version 3.4.0
-- Checking for module 'libnl-route-3.0'
--   Found libnl-route-3.0, version 3.4.0
-- Found GTest: /opt/deps/lib/cmake/GTest/GTestConfig.cmake (found version "1.11.0")
      .
      .
-- Checking for module 'libunbound'
--   No package 'libunbound' found
-- Found Unwind: /usr/include/x86_64-linux-gnu (found version "1.2.1")
-- Found Protobuf: /opt/deps/lib/libprotobuf.so;-lpthread (found version "3.20.3")
-- Found ZLIB: /opt/deps/lib/libz.so (found version "1.2.13")
-- Found OpenSSL: /usr/lib/x86_64-linux-gnu/libcrypto.so (found version "1.1.1f")
-- Found c-ares: /opt/deps/lib/cmake/c-ares/c-ares-config.cmake (found version "1.19.1")
      .
      .
-- Found Unwind: /usr/include/x86_64-linux-gnu (found suitable version "1.2.1", minimum required is "1.2.1")
-- Configuring done
-- Generating done
-- Build files have been written to: /home/dfoster/latest/build
```

Marking these symbols as ADVANCED limits the output of the `cmake -L` or `ccmake` commands to those variables that are inputs to the cmake build.

You can view all the symbols (including the advanced ones) via `cmake -LA` or the `t` command in `ccmake`.

**Review by 5abeel — APPROVED** [review]
LGTM

**Pull request merged**